server/dash/dashboards/ncSumComp.py

```diff
@@ -0,0 +1,407 @@
+import datetime
```

Comment:
please rename this file to be snake_case. Can you also spell it out more? I'm not sure what it's supposed to mean right now.

Reply:
Thanks for the suggestion - it stands for nc_summary_comparison